src/lib/Scenes/Inbox/Components/Conversations/ConversationCTA.tsx

```diff
@@ -13,21 +13,21 @@ interface Props {
 }
 
 export const ConversationCTA: React.FC<Props> = ({ conversation, show }) => {
-  // Determine whether we have a conversation about an artwork
-  const firstItem = conversation?.items?.[0]?.item
-  const artwork = firstItem?.__typename === "Artwork" ? firstItem : null
-  const artworkID = artwork?.artworkID
+  const liveArtwork = conversation?.items?.[0]?.liveArtwork
+  const artwork = liveArtwork?.__typename === "Artwork" && liveArtwork
```

Comment:
this `&& liveArtwork` seems unnecessary
```suggestion
  const artwork = liveArtwork?.__typename === "Artwork"
```

Reply:
Also, do we still need to check that it is typename Artwork, or are we just checking for whether there is a live artwork at all?

Here is another question. What if there is a snapshot but no live artwork because it is unpublished? Presumably the artwork is no longer offerable either but i wonder if this is a case where we need to tell the user something.

Reply:
I believe we do need to check `__typename` because the item can be of type `%other` or `null` and not doing so throws a type error. I can switch this conditional to be a ternary operator instead to get around this.

Re: no live artwork, I think in that case we wouldn't want to show Make Offer button. I would consider this the same as works that art not eligible or don't meet the requirements for inquiry make offer.